src/pug/components/mixin.pug

```diff
@@ -30,3 +30,17 @@ mixin rating (rate = 5)
     -const stars = [...Array(5)].map((_, index) => index < rate ? 'fa fa-star' : 'fa fa-star-half')
     each star in stars 
         i(class=star)
+
+mixin inputText(label,name,value,classLabel, classInput, isEdit,type)
+    label.form-label(class=classLabel for=name)= label
+    div(class=classInput)
+        if(isEdit)
+            input.form-control(type=type ? type:'text' id=name name=name value=value)
+        else 
+            p.desc=value
+
+mixin checkBox(label,name,classLabel)
+    div(class=classLabel)
+        .form-check 
+            input.form-check(type='checkbox' id=name name=name)
+            label.form-check-label.ps-2(for=name)=label
```

Comment:
Bọc input bên trong label là được. Không cần phải sử dụng tới 2 attr là for + id. Có thể áp dụng cho cả type radio hoặc type text, email thông thường
VD:
```
label.form-check-label.ps-2=label
    input.form-check(type='checkbox' name=name)
```